docs/graphql/core/deployment/deployment-guides/nhost-one-click.rst

```diff
@@ -23,43 +23,38 @@ One-click deploy on Nhost
 .. note::
    Make sure to `create an Nhost account <https://nhost.io/register>`__ first. Nhost offers a 30-day trial with its starter plan.
 
-Once you're logged into your Nhost account, click the button bellow to configure your project.
+Once you're logged into your Nhost account, click the ``create project`` button below to configure your project.
 
-.. image:: https://nhost.io/images/create-project-button.png
-   :width: 300px
-   :class: no-shadow
-   :alt: Deploy with Nhost
+.. thumbnail:: /img/graphql/core/deployment/create-project-nhost.png
```

Comment:
Can we use arrows in all screenshots to indicate what needs to be clicked?

Reply:
@marionschleifer Done.